go.mod

```diff
@@ -256,7 +256,9 @@ require (
 	rsc.io/tmplfunc v0.0.3 // indirect
 )
 
-replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101500.0-rc.3
+//replace github.com/ethereum/go-ethereum => github.com/ethereum-optimism/op-geth v1.101500.0-rc.3
+
+replace github.com/ethereum/go-ethereum => github.com/yuwen01/op-geth v1.0.2-operatorfee
```

Comment:
This PR is blocked by https://github.com/ethereum-optimism/op-geth/pull/388 right? It should land in `op-geth` and then a release needs to be cut from trunk. We can then update the tag here to point to that release to unblock this PR.

Reply:
Yes exactly, I've setup this dirty tag to be able to run CI.